### PR TITLE
Changed appearance and sortablility of table and website

### PR DESCRIPTION
**Pull request by btcspry**
- Converted whole site to bootstrap
- Added "GitHub" and "Getting Started" links in the menubar
- Included "sortable.js" for sorting the table, client side
        - Removed server-side table sorting

I hope you accept this pull request because it adds a user front-end to view the inner workings of this project, and it's way more presentable.

### CODE REVIEW

**Comment by chris-belcher** [comment]
Looks great.

**Comment by btcspry** [comment]
Thanks!  Can you merge it into your master, after your review of course.  You may want to change the background color to white, although it's up to you.

**Comment by chris-belcher** [comment]
I don't have much of an opinion right now of the color scheme, or anything gui user-facing.

**Comment by btcspry** [comment]
Could you just accept the pull request then!  With that, I'll contact the owner of JoinMarket.io and get that updated.

File: orderbook.html [hunk 1]
```diff
@@ -1,62 +1,101 @@
...
+			ul li {
+				/* float: left; */
+				display: inline;
+				padding: 0 5px 0 5px;
+				border-right: 1px solid #000;
+			}
+			ul li a {
+				padding 0.2em 1em;
+			}
+		</style>-->
```
Comment (haarts) [inline orderbook.html]: You can remove these commented out styles?

**Comment by btcspry** [comment]
They can be removed when the pull request is accepted.

**Comment by chris-belcher** [comment]
I would prefer the commented out parts be removed before merging.

I usually leave PRs open for a while so others can comment.

**Comment by btcspry** [comment]
I have removed the commented out parts, it is added to this pull request!

**Comment by chris-belcher** [comment]
You seem quite eager to contribute code, have you seen this wiki page about small projects I think are good https://github.com/chris-belcher/joinmarket/wiki/What-can-I-do-for-JoinMarket%3F

**Comment by btcspry** [comment]
I have, I'll work on one or two today actually!  Thanks

**Pull request merged**